src/components/Header/Navbar.tsx

```diff
@@ -82,6 +82,11 @@ export const Navbar: React.FC = () => {
         >
           My Hosts
         </PrimaryLink>
+        {permissions?.canEditAdminSettings && (
```

Comment:
Also I don't think this should be a navbar item. I think it should be hidden in the dropdown menu like it is in the legacy UI.